importer/tests/test_import.py

```diff
@@ -286,9 +286,10 @@ def get_counts():
         ).distinct()
         self.assertEqual(htmls.count(), 1)
         self.assertEqual(
-            [asset.asset for asset in htmls[0].static_assets.all()],
-            [
+            sorted([
```

Comment:
I don't know. I assume it was, and maybe still does on some machines. It just broke for me on a fresh installation. In any case, this is more explicit.